src/app/pages/send/send-crypto-asset-form/components/amount-field.tsx

```diff
@@ -31,6 +32,17 @@ function getAmountModifiedFontSize(props: GetAmountModifiedFontSize) {
     : maxFontSize;
 }
 
+interface Lerp {
```

Comment:
```suggestion
interface LinearInterpolation {
```

let's use verbose naming

Reply:
Done